readthedocs/projects/views/mixins.py

```diff
@@ -44,3 +47,46 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context[self.project_context_object_name] = self.get_project()
         return context
+
+
+class ProjectImportMixin:
+
+    """Helpers to import a Project."""
+
+    def import_project(self, project, tags, request):
```

Comment:
Maybe `finish_import_project` is a better name here, what do you think @ericholscher? Considering that the Project object is created outside this function and this one does the final steps.

Reply:
That seems better.